smithy-aws-protocol-tests/model/awsQuery/xml-structs.smithy

```diff
@@ -436,7 +436,7 @@ structure XmlNamespacesOutput {
 }
 
 // Ingored since it's not at the top-level
-@xmlNamespace(uri: "http://foo.com")
+@xmlNamespace(uri: "http://boo.com")
```

Comment:
Changed to show distinction from the also ignored `@xmlNamespace` trait on `XmlNamespacesOutput`.